home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1107,6 +1111,410 @@ paths:
           $ref: '#/components/responses/NotFound'
         default:
           $ref: '#/components/responses/InternalServerError'
+
+  /houses/{house_id}/apartments:
+    post:
+      tags:
+        - apartment
+      summary: "Create apartment"
+      description: Create new Apartment in the house
+      operationId: createApartment
+      parameters:
+        - name: "house_id"
+          in: "path"
+          description: "Use 1 for testing."
+          required: true
+          schema:
+            type: integer
+            format: int64
+            example: 1
+      requestBody:
+        description: Create a new Apartment
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateApartment'
+        required: true
+      responses:
+        '201':
+          description: Successfully created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ReadApartment'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        "403":
+          $ref: '#/components/responses/Forbidden'
+        default:
```

Comment:
What is about 404? I think it can appear here.

Reply:
> What is about 404? I think it can appear here.

Proposed changes have been made